firefox-ios/firefox-ios-tests/Tests/ClientTests/ImpressionTrackingUtilityTests.swift

```diff
@@ -0,0 +1,184 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+@testable import Client
+
+final class ImpressionTrackerTests: XCTestCase {
+    private var subject: ImpressionTrackingUtility!
+
+    override func setUp() {
+        super.setUp()
+        subject = ImpressionTrackingUtility()
+    }
+
+    override func tearDown() {
+        subject = nil
+        super.tearDown()
+    }
+
+    // MARK: - Helpers
+    private struct SendCapture {
+        var calls = 0
+        var payloads: [[IndexPath]] = []
+
+        mutating func record(_ ips: [IndexPath]) {
+            calls += 1
+            payloads.append(ips)
+        }
+    }
+
+    private func makeIP(_ item: Int, _ section: Int = 0) -> IndexPath {
```

Comment:
```suggestion
    private func makeIndexPath(_ item: Int, _ section: Int = 0) -> IndexPath {
```

nit; feel free to ignore haha